schemas/device/MRICoil.schema.tpl.json

```diff
@@ -0,0 +1,53 @@
+{
+  "_extends": "/core/schemas/products/device.schema.tpl.json",
+  "_type": "https://openminds.ebrains.eu/neuroimaging/MRICoil",
+  "properties": {
+    "activeElements": {
+      "_instruction": "If applicable, list the identifiers of the active coil elements or channels.",
+      "type": "string"
+    },
+    "coilGeometry": {
+      "_instruction": "Add the geometry of the coil (e.g., birdcage, loop, Helmholtz).",
+      "_linkedTypes": [
+        "controlledTerms:MRIRadiofrequencyCoilGeometry"
+      ]
+    },
+    "dimension": {
+      "_instruction": "Add the dimention of this coil",
+      "_linkedCategories": [
+        "mathematicalShapes"
+      ]
+    },
+    "elementCount": {
+      "_instruction": "Total number of coil elements.",
+      "minimum": 1,
+      "type": "integer"
+    },
+    "mountingType": {
+      "_instruction": "Add the coil mounting type (e.g., BuiltIn, External, Interventional, Wearable).",
```

Comment:
the examples should be lower case 

"Add the coil mounting type (e.g., built-in, external, interventional, wearable)."